sdk/nextjs/src/pages/bucketing.ts

```diff
@@ -1,6 +1,6 @@
 import { DevCycleUser, DVCPopulatedUser } from '@devcycle/js-client-sdk'
 import { generateBucketedConfig } from '@devcycle/bucketing'
-import { sseURlGetter } from '../server/ably'
+import { sseURlGetter } from '../server/ably.js'
```

Comment:
why does this need to be `.js`?

Reply:
cuz an ES module in Node requires every file import to end in the file extension, it no longer lets you omit it.

And this package has to be an ES module because server actions in app router need to be ESM

But the "pages" router uses Node's ESM resolver so for it to work in a pages app it needs the file extensions